db/migrate/20190912201728_add_rbac_seed.rb

```diff
@@ -0,0 +1,13 @@
+class AddRbacSeed < ActiveRecord::Migration[5.2]
+  def change
+    create_table :rbac_seeds do |t|
+      t.string :external_tenant
+      t.timestamps
+    end
+    add_index :rbac_seeds, :external_tenant
+
+    Tenant.all.each do |tenant|
+      RbacSeed.create!(:external_tenant => tenant.external_tenant)
```

Comment:
Yeah, that's the thought. There is no nice way to create a request from inside a migration. Also, once reworked, we'll only use the `RbacSeed` lookup internally ( it will not be exposed via endpoint. ) If a Tenant isn't seeded the UI will be aware ( and/or the API ) seeing they will receive a 403 looking up any Portfolio records.

Reply:
@syncrou If there are tenants in the DB that don't have the seeding done, would this mark it incorrectly as seeded. Shouldn't this be done after we have done a seeding from the controller. It can be built up incrementally so even if we have multiple tenants some seeded and some not seeded, we will add them to the RbacSeed only if some one has called seeding on it.